README.md

```diff
@@ -25,111 +26,115 @@ KubePlus takes an application Helm chart and wraps it in a Kubernetes API (CRD).
 
 ### Resource utilization
+
 KubePlus provides controls to set per-namespace resource quotas. It also monitors usage of CPU, memory, storage, and network traffic at the application instance level. The collected metrics are available in different formats and can be pulled into Prometheus for historical usage tracking.
 
 ### Upgrades
+
 A running application instance can be updated by making changes to the spec properties of the CRD instance and applying it.
 KubePlus will update that application instance (i.e. helm upgrade of the corresponding helm release). 
 A new version of an application can be deployed by updating the application Helm chart under the existing Kubernetes CRD or registering the new chart under a new Kubernetes CRD. If the existing Kubernetes CRD object is updated, KubePlus will update all the running application instances (helm releases) to the new version of the application Helm chart.
 
 ### Customization
+
 The spec properties of the Kubernetes CRD wrapping the application Helm chart are the fields defined in the chart’s values.yaml file. Application deployments can be customized by specifying different values for these spec properties.
 
 ## Demo
 
 https://github.com/cloud-ark/kubeplus/assets/732525/efb255ff-fc73-446b-a583-4b89dbf61638
 
+## Getting Started with an example
 
+Let’s look at an example of creating a multi-instance WordPress Service using KubePlus. The WordPress service provider goes through the following steps towards this on their cluster:
 
-## Getting Started 
+1. Clone the repository locally
 
-Let’s look at an example of creating a multi-instance WordPress Service using KubePlus. The WordPress service provider goes through the following steps towards this on their cluster:
+2. Create cluster or use an existing cluster. For testing purposes you can create a [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/) cluster:
 
-1) Create cluster (or use an existing cluster).
-   For testing purposes you can create a minikube cluster:
+   `minikube start`
 
-   ``$ minikube start ``
+   or
 
-2) Download KubePlus plugins and set up the PATH
-```
-  wget "https://github.com/cloud-ark/kubeplus/blob/master/kubeplus-kubectl-plugins.tar.gz?raw=true"
-  mv kubeplus-kubectl-plugins.tar.gz\?raw\=true kubeplus-kubectl-plugins.tar.gz
-  gunzip kubeplus-kubectl-plugins.tar.gz
-  tar -xvf kubeplus-kubectl-plugins.tar
-  export KUBEPLUS_HOME=`pwd`
-  export PATH=$KUBEPLUS_HOME/plugins:$PATH
-  kubectl kubeplus commands
-```
+   `kind create cluster`
```

Comment:
Nice!

We had not tested with kind yet. Did everything work okay with kind cluster?

Reply:
Yes, I tested it on `kind` cluster and everything was working.